### ISSUE TITLE
[DOC] Needs doc on observed and simulated metrics

### ISSUE THREAD

**Issue by tai** [GoogleChrome/lighthouse#9637]
**Summary**
I found Lighthouse reporting odd performance metric values, and these "odd" values seem to come from "simulated" values computed by lighthouse after actual measurement.

Currently, I'm forced to read through the source code under lighthouse-core/computed/metrics/, but can we have any document that clearly describes how these simulated values are computed from actual measured values?

I decided to use internal "observed*" metrics for my analysis, but still need to explain why Lighthouse reports different "simulated" values in its report and how those were computed. It'd be great if I can just point to some official URL.

**Comment by patrickhulce** [comment]
Yes there are many official docs linked from the docs folder that describe how this works. The engine that simulates metrics is called "Lantern" so anything you see referencing that is related to simulation.

tl;dr - observed values report the raw unthrottled metrics while simulated values report the metrics as if they were on a slow 4G network and a mediocre mobile device.

* [Lantern Design Doc](https://docs.google.com/a/chromium.org/document/d/1pHEjtQjeycMoFOtheLfFjqzggY8VvNaIRfjC7IgNLq0/edit?usp=sharing)
* [Lantern Deck](https://docs.google.com/presentation/d/1EsuNICCm6uhrR2PLNaI5hNkJ-q-8Mv592kwHmnf4c6U/edit?usp=sharing)
* [Metric Variability and Accuracy Analysis](https://docs.google.com/document/d/1BqtL-nG53rxWOI5RO0pItSRPowZVnYJ_gBEQCJ5EeUE/edit?usp=sharing)

**Comment by tai** [comment]
Thanks, while I do still need to look into the source for how the value is computed, page 10 of the [Lantern Deck](https://docs.google.com/presentation/d/1EsuNICCm6uhrR2PLNaI5hNkJ-q-8Mv592kwHmnf4c6U/edit?usp=sharing) is quite useful to explain how much difference can be seen in the values lighthouse reports.

Just FYI, I'm getting "odd" values for my "desktop mode" measurement. Since simulated condition of "desktop mode" measurement isn't far off from actual native environment, I was surprised to see lighthouse reporting way fast performance for some pages that are known to be slow (I run 1000+ LH runs to get statistical data). Internal "observed*" metrics gave us data that are much more in sync to our understanding of the site, hence the request.

**Comment by tai** [comment]
BTW, the "See the details" link on page 10 that seems to have more information is currently not open for access - it'd be nice if you can make it public as well.

**Comment by patrickhulce** [comment]
Thanks for the explanation! Do you have example URLs of these pages that are reported to be fast but observed to be slow? We would certainly like to make improvements where possible :)

As for the "see the details" link, IIRC it just points to the internal version of  [Metric Variability and Accuracy Analysis](https://docs.google.com/document/d/1BqtL-nG53rxWOI5RO0pItSRPowZVnYJ_gBEQCJ5EeUE/edit?usp=sharing).

**Comment by tai** [comment]
Sure, I can share you the URL as it is a public site. Here it is: http://search.icbc-ltd.com/SearchCenter/search.aspx?KeyWord=%E6%8A%95%E8%B3%87

I monitored this URL using [lr-desktop-config.js](https://github.com/GoogleChrome/lighthouse/blob/7715eda3c5debe85f1cb903606bea4a516d839db/lighthouse-core/config/lr-desktop-config.js) as configuration.

I did 1000+ runs of lighthouse in a cron with "lighthouse --config-path=desktop-config.js --quiet --save-assets $URL" to obtain the result. I'm extracting JSON and HAR data out from saved assets for my custom analysis. I'm essentially using lighthouse as a tool to obtain raw data, instead of using the report it generates.

**Comment by tai** [comment]
FYI: https://www.debugbear.com/blog/network-throttling-methods

It seems I was hit by example case #3 in above URL, where lighthouse simulator fails to count in TTFB of given page. It explains what I have seen, which lighthouse reported ~1000ms FMP time for above URL which has ~2500ms TTFB.

**Comment by patrickhulce** [comment]
Ah thanks @tai! Yes, the decreased reliance on TTFB is an intentional decision to mitigate variance. TTFB tends to fluctuate greatly depending on the host and origin/destination routing path, so we use the median TTFB of all requests to the origin. If you want to take into account those factors in your performance measurements, definitely use `--throttling-method=provided` when running LH which will convert all of the metric audit values to use the observed value.

You can learn more about the `provided` throttling method and alternatives in our [throttling documentation](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md#how-do-i-get-packet-level-throttling).
